include/glow/Graph/PlaceholderBindings.h

```diff
@@ -88,6 +88,9 @@ class PlaceholderBindings final {
   /// tensors.
   void clear();
 
+  /// Removes the Tensor backing Placeholder \p P;
```

Comment:
What happens if P is not in the map? 
/// If P is not in the map nothing happens. 

or 

/// P must be a valid placeholder in the map.